src/base-element.js

```diff
@@ -869,22 +869,14 @@ export class BaseElement {
   }
 
   /**
-   * Configures the supplied element to have a "fill content" layout. The
-   * exact interpretation of "fill content" depends on the element's layout.
-   *
-   * If `opt_replacedContent` is specified, it indicates whether the "replaced
-   * content" styling should be applied. Replaced content is not allowed to
-   * have its own paddings or border.
+   * See src/core/dom.js for full description.
    *
    * @param {!Element} element
    * @param {boolean=} opt_replacedContent
    * @public @final
    */
   applyFillContent(element, opt_replacedContent) {
-    element.classList.add('i-amphtml-fill-content');
-    if (opt_replacedContent) {
-      element.classList.add('i-amphtml-replaced-content');
-    }
+    applyFillContent(element, opt_replacedContent);
```

Comment:
Would it make sense for `applyFillContent` to be defined locally in this file?

Reply:
Still relevant, since we don't really want other code using this instead of `element.applyFillContent`. Including it in `core` kinda implies it's fine, whereas no one will go out of their way to import it from here (except the compiler)

Reply:
Sure, I'm fine with that